### DISCUSSION TITLE
what version of kicad?

### DISCUSSION THREAD

**Discussion by ogg1e** [mebs/CM4-NAS#15] [General]
The file extensions don't make sense for a current version of kicad (5.1.9).  They should be named CM4_NAS.pro instead of CM4_NAS.kicad_pro  for example. I can't open them in 5.1.9.

**Comment by ogg1e** [comment]
ohh... are you using the new nightlies (upcoming 6.0 version)?

**Reply by mebs** [reply]
Sorry I was away for the day! Yes, I had to use a nightly version of KiCad because this is what was used with the official CM4 IO board design. The .kicad_pro format doesn't seem to be supported by the stable releases.
